lhotse/recipes/gigaspeech.py

```diff
@@ -65,7 +66,13 @@ def parse_utterance(
         audio: Any,
         root_path: Path
 ) -> Optional[Tuple[Recording, List[SupervisionSegment]]]:
-    recording = Recording.from_file(root_path / audio['path'], recording_id=audio['aid'])
+    recording = Recording(id=audio['aid'],
+                          sources=[AudioSource(type='file',
+                                               channels=list(range(int(audio['channels']))),
+                                               source=f'{root_path}/{audio["path"]}')],
+                          num_samples=round(int(audio['sample_rate']) * Seconds(audio['duration']), ndigits=8),
+                          sampling_rate=audio['sample_rate'],
```

Comment:
For OPUS files you might want to "hardcode" 48kHz here, and add a line afterwards `recording = recording.resample(16000)`